src/content/basics/giant-swarm-operational-layers/index.md

```diff
@@ -0,0 +1,69 @@
+weight = 30
+categories = ["basics"]
++++
+
+# Giant Swarm Operational Layers
+
+A Giant Swarm installation has several operational layers, which depict a separation of concerns both on an operational as well as on a security level. In the following we will define the layers and explain the intended operational model.
+
+## Operational Layers
+
+We will go through the operational layers one by one from the bottom (infrastructure) to the top (user space) and explain the intended opreational model by defining (typical) users and permission levels. The layers are:
+
+1. [Infrastructure](#infrastructure)
+2. [Giant Swarm Control Plane](#controlplane)
+3. [Giant Swarm API](#giant-swarm-api)
+4. [User Space](#userspace)
+
+### Infrastructure {#infrastructure}
+
+The infrastructure layer covers the area on top of actual (or virtual) machines, networking, etc., which is managed by Giant Swarm SREs (Site Reliability Engineers) usually accessed through VPN and bastion hosts as well as through the cloud provider APIs if applicable.
+
+This layer does not include the actual hardware and maintenance of the data center. This is either covered by the (internal or external) data center provider or by the cloud provider.
+
+Giant Swarm SREs on this layer have root level SSH access to everything that pertains to a Giant Swarm installation. This is facilitated by a Single Sign On (SSO) mechanism including MFA (multi-factor authentication). On the cloud they additionally have access to the cloud account/subscription through a role to set up and manage the cloud resources.
+
+### Giant Swarm Control Plane {#controlplane}
+
+The Giant Swarm Control Plane consists mainly of services running inside the Control Plane Kubernetes cluster.
+
+Just like the former layer, this layer is accessed through VPN and bastion hosts. Giant Swarm SREs and operations personell have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
+
+### Giant Swarm API {#giant-swarm-api}
+
+The Giant Swarm API (Giant Swarm API) is a customer facing API that is usually whitelisted for only a certain IP range within the customer's network. This layer covers the API itself, but also its client manifestations in form of the Happa Web UI and `gsctl` CLI.
+
+On this layer there are two levels of access:
+
+#### 1. Giant Swarm API admin
+
+This access level is reserved for Giant Swarm operations and support personell and like the above layers facilitated by SSO with MFA.
+
+Such admin users have access to all organizations and all clusters in the Giant Swarm installation.
+
+#### 2. Giant Swarm API user
+
+This is the standard type of Giant Swarm API user that is given out to DevOps/Operations personell on the customer side. Usually that covers only few users that are tasked with cluster creation and management.
+
+Such users have access to all clusters in the organizations they belong to. They can be considered multi-cluster admins.
```

Comment:
Should we add here that these users can also create and delete organizations?

Reply:
yes, actually added a bit more here